README.md

```diff
@@ -37,7 +35,23 @@ addItem item =
         |> withHeader "X-My-Header" "Some Header Value"
         |> withJsonBody (itemEncoder item)
         |> withTimeout 10000
-        |> withExpectJson itemsDecoder
+        |> withExpectJson GotItem itemsDecoder
         |> withCredentials
-        |> send handleRequestComplete
+        |> send
+
+type Msg = GotItem (Result Http.Error (List String))
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    case msg of
+        GotItem (Ok items) ->
+          ( { model | items = items }
+          , Cmd.none
+          )
+
+        GotItem (Err err) ->
+          -- Handle the error...
+          let _ = Debug.log (Debug.toString err)
```

Comment:
Isn't this debug statement going to prevent this package from being compiled with --optimize?

Reply:
Hi @koenusz 

Thanks for reviewing.
I updated the README and the example.

Regards
Dieter